README.md

```diff
@@ -131,6 +131,18 @@ class AlbumRepresenter < Representable::Decorator
 end
 ```
 
+## Formats
+
+The gem supports JSON, XML, YAML, hashes and Struct-based objects for the representing output. 
+To use a specific format, include the corresponding module in your representer and use `to_<format>` method on a represented object:
+
+- `Representable::JSON#to_json`
+- `Representable::JSON#to_hash` (provides a hash instead of string)
+- `Representable::Hash#to_hash`
+- `Representable::Object#to_object` (provides a Struct-based object)
```

Comment:
then it should be to_struct.

Reply:
What if we make the object type configurable and `Struct` is one of the possible targets?

Reply:
I like this. But having a to_struct shortcut is good too.

Reply:
I agree, `module Struct` and `to_struct` fit more, as `to_object` is too vague. Moved the new code to a new module.

Frankly, I'm kinda confused about `configurable object type` with Struct as one of the possible targets. What other possible target there could be..? OpenStruct / AnyCustomUserClassPassedAsArgument? I feel like Struct is sufficient and there's no need for other options

Reply:
Object can return the original object such as ActiveRecord model or a Poro with it attributes set.

Reply:
> Frankly, I'm kinda confused about `configurable object type` with Struct as one of the possible targets. What other possible target there could be..?

Imagine a project with `Domain::Song`, `Domain::Song::Duration`, `Domain::Artist`, etc. People might want to transform an object to another domain object?!